presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -13,14 +13,14 @@ class AnonymizerEngine:
     and replaces the PII entities with the desired transformations.
     """
 
+    def __get_sub_classes():
+        sub_classes = {}
+        for cls in Anonymizer.__subclasses__():
+            sub_classes[cls.__name__.lower()] = cls
```

Comment:
Not critical, you can also use comprehension: 
{cls.__name__.lower(): cls for cls in Anonymizer.__subclasses__()}